### PR TITLE
Inject codesnippets for readme, update changelog Text Analytics

### LINKED ISSUE #6503: Release Text Analytics January preview
- Confirm and update pom's to reflect the agreed package version.
- Update readmes and samples with the updated version.
- Update changelog
- Add text analytics in Java release notes.
- Release Text Analytics preview.
- Add #7020

### PR DESCRIPTION
**Pull request by samvaity**
Fixes #6503, #7153, #6961

### CODE REVIEW

File: sdk/textanalytics/azure-ai-textanalytics/src/samples/java/com/azure/ai/textanalytics/ReadmeSamples.java [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+    public void useSubscriptionKeySyncClient() {
+        TextAnalyticsClient textAnalyticsClient = new TextAnalyticsClientBuilder()
+            .subscriptionKey(SUBSCRIPTION_KEY)
+            .endpoint(ENDPOINT)
+            .buildClient();
+    }
+
+    /**
+     * Code snippet for getting async client using subscription key authentication.
+     *
```
Comment (mssfang) [inline sdk/textanalytics/azure-ai-textanalytics/src/samples/java/com/azure/ai/textanalytics/ReadmeSamples.java]: extra lines for all other methods as well

File: sdk/textanalytics/azure-ai-textanalytics/README.md [hunk 2]
```diff
@@ -83,10 +83,10 @@ For starters, by having the Netty or OkHTTP dependencies on your classpath, as s
 
 <!-- embedme ./src/samples/java/com/azure/ai/textanalytics/ReadmeSamples.java#L158-L161 -->
 ```java
-HttpClient client = new NettyAsyncHttpClientBuilder()
-    .port(8080)
-    .wiretap(true)
-    .build();
+public void configureHttpClient() {
```
Comment (mssfang) [inline sdk/textanalytics/azure-ai-textanalytics/README.md]: you need to update the line number as well

File: sdk/textanalytics/azure-ai-textanalytics/README.md [hunk 3]
```diff
@@ -176,12 +176,12 @@ analyze sentiment, recognize entities, detect language, and extract key phrases
...
 your resource and a subscription key that allows you access:
 
-<!-- embedme ./src/samples/java/com/azure/ai/textanalytics/ReadmeSamples.java#L29-L32 -->
+<!-- embedme ./src/samples/java/com/azure/ai/textanalytics/ReadmeSamples.java#L28-L31 -->
 ```java
-TextAnalyticsClient textAnalyticsClient = new TextAnalyticsClientBuilder()
-    .subscriptionKey(SUBSCRIPTION_KEY)
-    .endpoint(ENDPOINT)
-    .buildClient();
+public void useSubscriptionKeySyncClient() {
```
Comment (mssfang) [inline sdk/textanalytics/azure-ai-textanalytics/README.md]: wrong line number

File: sdk/textanalytics/azure-ai-textanalytics/README.md [hunk 4]
```diff
@@ -248,24 +248,25 @@ The following sections provide several code snippets covering some of the most c
...
 `TextAnalyticsClientBuilder`,
 
-<!-- embedme ./src/samples/java/com/azure/ai/textanalytics/ReadmeSamples.java#L29-L32 -->
+<!-- embedme ./src/samples/java/com/azure/ai/textanalytics/ReadmeSamples.java#L28-L31 -->
 ``` java
-TextAnalyticsClient textAnalyticsClient = new TextAnalyticsClientBuilder()
-    .subscriptionKey(SUBSCRIPTION_KEY)
-    .endpoint(ENDPOINT)
-    .buildClient();
+public void useSubscriptionKeySyncClient() {
```
Comment (mssfang) [inline sdk/textanalytics/azure-ai-textanalytics/README.md]: wrong line number

Comment (mssfang) [reply sdk/textanalytics/azure-ai-textanalytics/README.md]: there are more. But I stop adding comments here

Comment (samvaity) [reply sdk/textanalytics/azure-ai-textanalytics/README.md]: should be updated now :)

**Pull request merged**